module-1/lab-data_cleaning/main.ipynb

```diff
@@ -0,0 +1,6760 @@
+{
```

Comment:
1. Good for using display method to get a more comprehensive dataframe look&feel. When you need to print an output involving a dataframe use the head () method, this facilitates your comprehension and also the people you share the file with. 
2. When you are importing data from a database, it is common to check with code if the number of records and the field data types you are importing match the ones in the database server. 
3. I see some comments on your notebook saying "no entiendo que es lo que ha pasado con.....", please take note of your questions and doubts and get back to me. I'll try to do my best for you to understand.
4. Step 10: When you need to make a decision on whether or not is recommended to drop missing values, think about the total number of records we have in our dataset. For example, if we have a dataset with 50.000 records and we have more than the 50% of missing values for a certain variable, it's not a good idea to drop them because we can have some problems of insuficient data for further modeling stage. The reason it's not that I have other columns that interest to my analysis.... We can review a couple of issues here when you have some time. 
5. Step 11: There are a couple of interesting doubts you have arised that I think we could get through in tomorrow's exercise review. Congrats for saying I don't know, I don't understand.... That's the way!!.